doc/Learn/Hosting/HowTo-HostingSetup.md

```diff
@@ -6,7 +6,7 @@ uid: Uno.Extensions.Hosting.HowToHostingSetup
 `Uno.Extensions.Hosting` can be used to register services that will be accessible throughout the application via dependency injection (DI). This tutorial will walk you through the critical steps needed to leverage hosting in your application.
 
 > [!WARNING]
-> The steps outlined here are unnecessary if you used the new project wizard template to create your solution. Otherwise, it is recommended that you follow the [Creating an application with Uno.Extensions article](xref:Uno.Extensions.HowToGettingStarted) for creating an application from the template.
+> The steps outlined here are unnecessary if you used the new project wizard template to create your solution. Otherwise, it is recommended that you follow the [Creating an app article](xref:Uno.GettingStarted.CreateAnApp.VS2022) for creating an application from the template.
```

Comment:
```suggestion
> The steps outlined here are unnecessary if you use the new project wizard template to create your solution. Otherwise, it is recommended that you follow the [Creating an app article](xref:Uno.GettingStarted.CreateAnApp.VS2022) for creating an application from the template.
```

Reply:
@agneszitte I think it hould be "used" here, because it's talking about an existing Uno application, in this case, ins't it?